demo/lightning/text-based/distributed/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Base Image: Start with a slim Python image that already includes Python3.
```

Comment:
In order to simplify things, you might want to make this work with the Dockerfile in the base of the repo. That also has the benefit of packaging up the whole repo including local changes, so that you can make and test changes without pushing them to PyPi

Reply:
Yes i'll update that alongside the readme that includes instructions on how to run